src/xunit.performance.api/EtwPerformanceMetricEvaluationContext.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Parsers.Kernel;
+using Microsoft.Diagnostics.Tracing.Parsers.MicrosoftXunitBenchmark;
+using Microsoft.Xunit.Performance.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal class EtwPerformanceMetricEvaluationContext : PerformanceMetricEvaluationContext, IDisposable, IPerformanceMetricReader
```

Comment:
Almost a straight copy from the *run project (net46). I believe we should break down projects and do general cleanup, but that should be outside this PR. At the same time, it might be good to have a class we can extend without worrying about breaking "legacy code"?